Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -11,7 +11,7 @@
 //===----------------------------------------------------------------------===//
 
 @_spi(RawSyntax) import SwiftSyntax
-@_spi(Testing) @_spi(RawSyntax) import SwiftParser
+@_spi(Testing)@_spi(RawSyntax) import SwiftParser
```

Comment:
Same as previous comment

Reply:
I think these changes are made by the formatter. Already discard them.